CODEOWNERS

```diff
@@ -7,6 +7,7 @@
 /pkg/rpm/		@nacl
 /pkg/tests/rpm/		@nacl
 /legacy/tests/rpm	@nacl
+/toolchains/rpm		@nacl
```

Comment:
It's nice that github tells you this is valid.

Reply:
I'm not sure I understand the comment.

Reply:
The review UI says this, at least for me:

<img width="342" alt="image" src="https://user-images.githubusercontent.com/70147/154163832-775b0e27-9c4a-4ffa-bbda-e5102abb483c.png">